.github/workflows/publish.yaml

```diff
@@ -0,0 +1,25 @@
+name: Publish gem to Rubygems
+on:
+  push:
+    branches:
+      - master
```

Comment:
We probably don't wan't main/master because we probably want the tag to be the version released

Reply:
Updated to match all branches. Test rubygems publish succeeded: https://rubygems.org/gems/authzed/versions/0.0.1